docs/10_utilities/index.md

```diff
@@ -5,5 +5,4 @@ link_text: Antelope Utilities
 
 This section contains documentation for additional utilities that complement or extend `nodeos` and potentially other Antelope software:
 
```

Comment:
Not in the scope of this PR, but here probably where user would expect link to separate md file describing leap-util, I feel like with removal of eosio-blocklog.md users loose valuable guide, it just became a subcommand in leap-util now.